metals/src/main/scala/scala/meta/internal/metals/SlowTaskConfig.scala

```diff
@@ -1,15 +0,0 @@
-package scala.meta.internal.metals
-
-final case class SlowTaskConfig(value: String) {
-  def isOff: Boolean = value == "off"
-  def isOn: Boolean = value == "on"
-  def isStatusBar: Boolean = value == "status-bar"
```

Comment:
I didn't know this existed before. AFAIK, no clients use this. Is it safe to assume I can just remove this, and the remove the following check in `ConfiguredLanguageClient.scala`:

```scala
      if (config.slowTask.isStatusBar) {
        metalsStatus(MetalsStatusParams(params.message))
      }
```

Reply:
Fine by me 👍